### DISCUSSION TITLE
Why Are a lot of features locked?

### DISCUSSION THREAD

**Discussion by Drewster65** [supermerill/SuperSlicer#3423] [General]
I want to change the wall thickness to say 0.4 to calibrate flow rate but the wall feature is locked and I can't seem to find out why I am unable to enter a value.
anybody able to help with this please

**Comment by neophyl** [comment]
First of all what version are you using ?  There are many changes between a 2.3/2.4/2.5 version.
Also there are lots of possibilities.  This is why we ask for a zipped up 3mf project files saved from the slicer to be attached.  With a project file we can examine all the settings and have a much much greater chance of being able to give an answer.

File>save Project as to save the 3mf and then ZIP it up or the site wont let it be attached.
